explore/src/main/scala/explore/tabs/TargetTabContents.scala

```diff
@@ -167,6 +167,13 @@ object TargetTabContents extends TwoPanels:
         )
     )
 
+    def getObsTime(idsToEdit: ObsIdSet): ProgramSummaries => Option[Instant] = a =>
```

Comment:
Why move this here? It doesn't seem to be called from anywhere else.

Reply:
I was going to share it with another function but I didn't need it after all